payment_cycle/apps.py

```diff
@@ -1,6 +1,31 @@
 from django.apps import AppConfig
 
+MODULE_NAME = 'payment_cycle'
+
+DEFAULT_CONFIG = {
+    'gql_query_payment_cycle_perms': ['200001'],
+    'gql_mutation_process_payment_cycle_perms': ['200005']
```

Comment:
Perms are different in migration and apps.py, lets stick to those in migration. I already updated doc page with rights from migration.

Reply:
yes, I raised this point above already @jdolkowski